packages/types/lib/index.ts

```diff
@@ -1,30 +1,31 @@
-import BigNumber from 'bignumber.js'
-import { SwapProvider, SwapParams } from './swap'
-import { ChainProvider, SendOptions } from './chain'
-import { WalletProvider } from './wallet'
+import BigNumber from "bignumber.js";
+import { SwapProvider, SwapParams } from "./swap";
+import { ChainProvider, SendOptions } from "./chain";
+import { WalletProvider } from "./wallet";
 
-import { FeeProvider, FeeDetails, FeeDetail } from './fees'
-import { Transaction } from './transaction'
-import { Block } from './block'
-import { Address } from './address'
-import { Network } from './network'
+import { FeeProvider, FeeDetails, FeeDetail } from "./fees";
+import { Transaction } from "./transaction";
+import { Block } from "./block";
+import { Address } from "./address";
+import { Network } from "./network";
```

Comment:
some weird formatting going on here

Reply:
a prettier configuration should fix this.